modules/Link.js

```diff
@@ -119,7 +118,7 @@ const Link = React.createClass({
       if (activeClassName || (activeStyle != null && !isEmptyObject(activeStyle))) {
         if (router.isActive(location, onlyActiveOnIndex)) {
           if (activeClassName)
-            props.className += props.className === '' ? activeClassName : ` ${activeClassName}`
+            props.className = `${props.className || ''} ${activeClassName}`.trim()
```

Comment:
Is there any harm to just pulling in `classnames` here? I can't imagine anybody doesn't have `classnames` already installed, so this should dedupe trivially.

Otherwise IMO this is better written as:

```js
if (activeClassName) {
  if (props.className) {
    props.className += ` ${activeClassName}`
  } else {
    props.className = activeClassName
  }
}
```